engine/flink/kafka-components-utils/src/main/scala/pl/touk/nussknacker/engine/kafka/source/flink/FlinkKafkaSource.scala

```diff
@@ -72,12 +81,63 @@ class FlinkKafkaSource[T](
 
   protected lazy val topics: NonEmptyList[TopicName.ForSource] = preparedTopics.map(_.prepared)
 
+  private val defaultOffsetResetStrategy =
+    if (kafkaConfig.forceLatestRead.contains(true)) OffsetResetStrategy.Reset else OffsetResetStrategy.Continue
+
+  override def actionParametersDefinition: Map[ScenarioActionName, Map[ParameterName, ParameterConfig]] = {
+    Map(
+      ScenarioActionName.Deploy -> Map(
+        OFFSET_RESET_STRATEGY_PARAM_NAME -> ParameterConfig(
+          defaultValue = Some(defaultOffsetResetStrategy.toString),
+          editor = Some(
+            FixedValuesParameterEditor(
+              List(
+                FixedExpressionValue(
+                  OffsetResetStrategy.Continue.toString,
+                  s"Resume reading data where it previously stopped (continue)."
+                ),
+                FixedExpressionValue(
+                  OffsetResetStrategy.Reset.toString,
+                  "Start reading new events only (reset)."
+                ),
+                FixedExpressionValue(
+                  OffsetResetStrategy.Restart.toString,
+                  "Rewinds reading from the earliest event (restart)."
+                ),
+              ),
+              mode = FixedValuesEditorMode.RADIO
+            )
+          ),
+          validators = None,
+          label = Some("Offset reset strategy"),
```

Comment:
How about "Reset offsets", with descriptions:
- No, continue from the last known position
- Yes, read only events added after this scenario is deployed
- Yes, read all events from the topic

Reply:
I've chosen similar descriptions :) Please check if it looks better. Some guys insisted on "strategy" in the name of the parameter (it sounds technically correct, but might be obscure for some people).

Reply:
Shouldn't we use the word "messages" here, instead of "events"? 

And the "Start" word is IMO superfluous, we can just say "Read new events (-> messages) only"?